app/mailers/group_smtp_mailer.rb

```diff
@@ -15,7 +15,9 @@ def send_mail(from_group, to_address, post, cc_addresses: nil, bcc_addresses: ni
       domain: from_group.email_username_domain,
       user_name: from_group.email_username,
       password: from_group.email_password,
-      authentication: GlobalSetting.smtp_authentication,
+      # NOTE: Might be better at some point to store this authentication method in the database
+      # against the group.
+      authentication: SmtpProviderOverrides.authentication_override(from_group.smtp_server),
```

Comment:
I assume this is the functional change and the rest is lift-and-shift?

Reply:
Yep correct!